testing/unit/tp/node_account/shared.py

```diff
@@ -0,0 +1,48 @@
+from sawtooth_sdk.protobuf.setting_pb2 import Setting
+
+from testing.mocks.stub import StubContext
+from remme.tp.node_account import NodeAccountHandler, NodeAccount
+
+RANDOM_NODE_PUBLIC_KEY = '039d6881f0a71d05659e1f40b443684b93c7b7c504ea23ea8949ef5216a2236940'
+
+
+NODE_ACCOUNT_ADDRESS_FROM = 'c2a4acd71fa7e120c60fb392a64fd69de891a60c667d9ea9e5d9d9d617263be6c20202'
+SETTINGS_ADDRESS = '0000007ca83d6bbb759da9cde0fb0dec1400c54773f137ea7cfe91e3b0c44298fc1c14'
```

Comment:
`SETTINGS_ADDRESS` to get what? 

`GET_MINIMUM_STAKE_SETTINGS_ADDRESS`, for instance.

Reply:
Done. MINIMUM_STAKE_SETTINGS_ADDRESS.